solc_select/solc_select.py

```diff
@@ -66,12 +101,24 @@ def is_older_linux(version):
     )
 
 
+def is_older_windows(version):
+    return soliditylang_platform() == "windows-amd64" and StrictVersion(version) <= StrictVersion(
+        "0.7.1"
+    )
+
+
 def get_url(version, artifact):
     if is_older_linux(version):
         return f"https://raw.githubusercontent.com/crytic/solc/master/linux/amd64/{artifact}"
     return f"https://binaries.soliditylang.org/{soliditylang_platform()}/{artifact}"
 
 
+def get_artifact(version, artifact_file_dir):
```

Comment:
this function is unused, is this intentional?

Reply:
Good catch! Removed in d583428